### PR TITLE
Goldrain auto Clicker

### PR DESCRIPTION
**Pull request by almidar**

### CODE REVIEW

File: autoPlay.js [hunk 1]
```diff
@@ -50,7 +55,7 @@ if (thingTimer){
 function firstRun() {
 	// disable particle effects - this drastically reduces the game's memory leak
 	if (g_Minigame !== undefined) {
-		g_Minigame.CurrentScene().DoClickEffect = function() {};
+	//	g_Minigame.CurrentScene().DoClickEffect = function() {}; //this is irritating
```
Comment (mouseas) [inline autoPlay.js]: This is in place to minimize the game's horrible memory leak. By all means disable it in your version, but the main branch is going to leave this in.

File: autoPlay.js [hunk 2]
```diff
@@ -12,7 +12,12 @@
 // IMPORTANT: Update the @version property above to a higher number such as 1.1 and 1.2 when you update the script! Otherwise, Tamper / Greasemonkey users will not update automatically.
 
 var isAlreadyRunning = false;
-
+//vvvvvvvvvvv Variables for Goldrain Clicker
```
Comment (mouseas) [inline autoPlay.js]: Can you please leave off these separator comments?

**Comment by win0na** [comment]
It looks like there are still merge conflicts, however, GitHub may have just failed to update it on my end.
ss 2015-06-13 at 01 15 31

**Comment by mouseas** [comment]
@almidar can you merge in the latest version and fix your merge conflicts? They're still pretty significant.

**Comment by win0na** [comment]
Said conflicts have been fixed.

**Pull request merged**